web/src/Routes.tsx

```diff
@@ -1,6 +1,7 @@
 import { Router, Route, Set } from '@redwoodjs/router'
 
 import SidebarLayout from 'src/layouts/SidebarLayout'
+import GraphQLSchemaPage from 'src/pages/GraphQLSchemaPage/GraphQLSchemaPage'
```

Comment:
Remove please. Are VSCode not picking up on the new pages you create? Are you running with `yarn rw dev`?